src/analyses/reaching_definitions.cpp

```diff
@@ -827,8 +829,8 @@ bool rd_range_domaint::merge_shared(
   {
     const irep_idt &identifier=value.first;
 
-    if(!ns.lookup(identifier).is_shared() /*&&
-       !rd.get_is_dirty()(identifier)*/)
+    if(!ns.lookup(identifier).is_shared()
+       /*&& !rd.get_is_dirty()(identifier)*/)
```

Comment:
I'm going to address this in a separate PR - the related TODO is a few lines above this commented bit.